### PR TITLE
issue #160 solved

### PR DESCRIPTION
**Pull request by adamovanja**
Solving issue trim_right qiime2/q2-feature-classifier#160

### CODE REVIEW

**Comment by adamovanja** [comment]
Hi @nbokulich,
I have adjusted the code style to be more pep8 compliant by checking for errors using pylint, as ```make lint``` returned an error.
Also, I adjusted the order of the trim and trunc options to ```trim_right```, ```trunc_len```, ```trim_left```. If your colleagues suggest the former order, I am happy to revert the last commit.

File: q2_feature_classifier/_cutter.py [hunk 1]
```diff
@@ -175,11 +181,14 @@ def extract_reads(sequences: DNASequencesDirectoryFormat, f_primer: str,
...
         containing the reads
     """
-    if min_length > trunc_len - trim_left and trunc_len > 0:
+    if min_length > trunc_len - (trim_left + trim_right) and trunc_len > 0:
         raise ValueError('The minimum length setting is greater than the '
                          'length of the truncated sequences. This will cause '
                          'all sequences to be removed from the dataset. To '
-                         'proceed, set a min_length ≤ trunc_len - trim_left.')
+                         'proceed, set '
+                         '"min_length ≤ trunc_len - (trim_left  + '
```
Comment (nbokulich) [inline q2_feature_classifier/_cutter.py:189]: looks like there is an extra " character on this line, probably residue from earlier changes... could you please remove?

**Comment by adamovanja** [comment]
Hi @nbokulich,
Sure, I performed the changes. I now also installed q2lint, which did not raise any issues locally.

**Comment by thermokarst** [comment]
>     2\. so I just want to poll @BenKaehler @thermokarst @ebolyen to see if they have any objections

No objections here.

**Pull request merged**